tests/misc_env/install_prereq.py

```diff
@@ -42,15 +42,18 @@ def run(**kw):
     repo = config.get("add-repo", False)
     rhbuild = config.get("rhbuild")
 
+    is_live = config.get("is_live", False)
```

Comment:
"live" is abiguous to me. Can we rename this setting to `is_production` instead? Is this "live" name used anywhere else (in any internal documentation, etc)?

Reply:
I agree that 'live' can be replaced with 'production' to make it unambiguous.